### ISSUE TITLE
Some... quality control issues?

### ISSUE THREAD

**Issue by MrShigura** [Audio-Injector/stereo-and-zero#6]
Apologize if I'm misunderstanding something, but... your site shows this...

![stereoa](https://user-images.githubusercontent.com/37914432/38348402-5956d4f8-3856-11e8-801b-78589356d5aa.jpg)

![audioinjector pi soundcard side](https://user-images.githubusercontent.com/37914432/38348692-2178899e-3858-11e8-9621-096b6ae7096e.jpg)

What I got was this...

![img_2121](https://user-images.githubusercontent.com/37914432/38348855-331132e0-3859-11e8-89a5-beec5c9d9ea6.jpg)

No hammer header like the one in the product picture, and with the pin cover that did come with it off, the gpio pins are kind of doing the wave...

![img_2135](https://user-images.githubusercontent.com/37914432/38348889-50299e94-3859-11e8-99bc-7fd6591e7e1d.jpg)

Also one of the rca jacks and the knob next to it are on crooked...

![img_2133](https://user-images.githubusercontent.com/37914432/38348978-ca3988b6-3859-11e8-9f73-005b8bc669e6.jpg)

Again sorry if this was the wrong place to post this, but I didn't see a way to contact you directly and this seems a bit off...

**Comment by flatmax** [comment]
I understand that whilst the functionality is equal to the picture, 
aesthetic is slightly different.

You will notice that the prototype's horrible through hole capacitors 
have been replaced with surface mount and various other things have 
changed for the better.

If those aesthetics really trouble you, then I encourage you to return 
your item for a refund.

**Comment by MrShigura** [comment]
Fair enough, as long as it's as intended, not really an issue, was just making sure before I proceeded, again, sorry for the trouble.
